<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JsOptions</class>
 <widget class="QDialog" name="JsOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JavaScript</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>Filtering</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;b&gt;Site whitelist&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Javascript on these servers will ALWAYS be turned on (even if you have disabled javascript globaly)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="jsWhiteAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="jsWhiteRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QListWidget" name="jswhiteList"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;b&gt;Site blacklist&lt;/b&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Javascript on these servers will NEVER be turned on</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QListWidget" name="jsblackList"/>
       </item>
       <item row="5" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_1">
         <item>
          <spacer name="verticalSpacer_1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="jsBlackAdd">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="jsBlackRemove">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QDialogButtonBox" name="close">
         <property name="standardButtons">
          <set>QDialogButtonBox::Close</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Allow JavaScript to:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="jscanCloseWindow">
           <property name="text">
            <string>Close windows</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="jscanOpenWindow">
           <property name="text">
            <string>Open popup windows</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="jscanChangeSize">
           <property name="text">
            <string>Change window size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="jscanHideMenu">
           <property name="text">
            <string>Hide menu bar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="jscanHideStatus">
           <property name="text">
            <string>Hide status bar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="jscanHideTool">
           <property name="text">
            <string>Hide tool bar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="jscanAccessClipboard">
           <property name="text">
            <string>Access clipboard</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="close2">
         <property name="standardButtons">
          <set>QDialogButtonBox::Close</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
 </connections>
</ui>
